Dockerfile.cross

```diff
@@ -1,13 +1,17 @@
 # This image is meant to enable cross-architecture builds.
 # It assumes the foundry binaries have already been compiled for `$TARGETPLATFORM` and are
 # locatable in `./dist/bin/$TARGETARCH`
-FROM ubuntu:22.04
+FROM alpine:3.20
 
 # Filled by docker buildx
 ARG TARGETARCH
 
+RUN apk add --no-cache git
+
 COPY ./dist/bin/$TARGETARCH/* /usr/local/bin/
 
+RUN adduser -Du 1000 foundry
```

Comment:
I remember having to do something like this a while ago on something else

Reply:
yeah, for this case is for being able to map host user to container user so there's no issue with volumes mapped permissions (resulting in git failure `fatal: detected dubious ownership in repository`)